### ISSUE TITLE
Multiple links around the web lead to "https://www.vttassets.com/asset/..." not "/assets/..." resulting in Page not Found

### ISSUE THREAD

**Issue by LaytonGB** [VTTAssets/vtta-tokenizer#7]
Downloading as a ZIP and extracting into a module didn't seem to install this correctly, and the forum post is gone so there's no alternate download. Is support continuing? If not, I'd appreciate that being posted. 
Thanks.

**Comment by LaytonGB** [comment]
Turns out the Virtual Table Top assets site was not down, but that if one is not logged in, one gets a page that says "page not found" when visiting actual pages on the site. Issue name adjusted accordingly.

**Comment by VTTAssets** [comment]
Sorry for getting so late to the party, I still rely mainly on Discord for reporting bugs instead of looking at user-generated issues (which is my fault, really). are you still having troubles?

**Comment by LaytonGB** [comment]
Yes, the link to your work on this page: https://www.dndbeyond.com/forums/d-d-beyond-general/general-discussion/53553-not-another-chrome-extension-foundry-vtt
is incorrect. I appreciate that this may be an issue you can no longer fix. Sorry for my delayed response.
